pkg/querier/chunk_store_queryable_test.go

```diff
@@ -15,6 +15,9 @@ import (
 	promchunk "github.com/cortexproject/cortex/pkg/chunk/encoding"
 )
 
+// Make sure that chunkSeries implements SeriesWithChunks
```

Comment:
If we are using `chunkSeries` as `SeriesWithChunks`, and if it is not compatible, it would be caught during build time itself. Do we need this additional test?

Reply:
Btw, never thought it could be tested like this :) TIL

Reply:
We aren't. We only check if series implements this interface, and if it does, then we use chunk-based code.

https://github.com/cortexproject/cortex/blob/fbe1c63a1f9f42963b93af865c35065f8ba30bde/pkg/querier/querier.go#L267

Reply:
Makes sense then